File: 268-missing-number/missing-number.c
int missingNumber(int* nums, int n){
    int sum = 0;
    if (n % 2 == 0)
    {
        sum = (n + 1) * (n / 2);
    }
    else sum = (n + 1) / 2 * n;
    for (int i = 0; i < n; i++)
    {
        sum -= nums[i];
    }
    return sum;
}

File: 268-missing-number/missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        int count = 0;
        int arrSize = nums.length;
        for (int i = 0; i < arrSize; i++)
        {
            count += nums[i];
        }
        return arrSize * (arrSize + 1) / 2 - count;
    }
}